src/services/appointmentData.service.js

```diff
@@ -93,8 +93,16 @@ function transformData(data) {
     // We are going to show a consolidated "Preregistration" card instead.
     // There is still code to display these sites individually in
     // Availability.js and SignupLink.js if we decide to go that way later on.
+    // ALSO.. Filter out all pharmacies
     mappedData = mappedData.filter((d) => {
-        return !d.isMassVax;
+        return (
```

Comment:
Should we instead:
1) stop scraping those sites & push the change, then
2) delete the parent locations so they aren't passed through to the front-end
?

Reply:
Here's my thought...

1. Merge this PR now
2. Turn off scrapers ( I did Walgreens already in a PR on the back-end )
3. Change the database retrieval code to not return these 4 sites at all
4. New PR to remove this code that is filtering on the front-end